db/migrate/20160520200712_recreate_user_groups.rb

```diff
@@ -0,0 +1,10 @@
+class RecreateUserGroups < ActiveRecord::Migration
+  def change
+    create_table :user_groups do |t|
+      t.string :name
```

Comment:
Should this be null: false?

I am a huge stickler for db constraints. I think mission reference can also be non null too right, since you can't have standard-mode user groups?